src/components/10-atoms/toggle-switch/README.md

```diff
@@ -14,6 +14,10 @@ If `active` is defined when first set under React, controlled-component mode is
 
 The Boolean attribute `disabled`, when true, disables the toggle switch natively.
 
+### error
+
+The String attribute `error` sets an error text. Leave `error` empty, if you don't want an error text.
```

Comment:
`string` not `String`.

Boolean is only written in uppercase because some that is the name of the one who found it. Ask markus for details. I don't really like it. :D

Reply:
In "professional english", you don't use shortcuts like `don't`. You always write it out: `do not`

`I am` instead of `I'm`
`He was not` instead of `He wasn't`

And so on. :)